addons/account/models/account_move.py

```diff
@@ -2288,8 +2312,22 @@ def _get_computed_uom(self):
             return self.product_id.uom_id
         return False
 
+    @api.multi
+    def _get_price_total_and_subtotal(self, price_unit=None, quantity=None, discount=None, currency=None, product=None, partner=None, taxes=None, move_type=None):
```

Comment:
Remove the `api.model` behavior from this `api.multi` method :)
Use the method below instead.

Reply:
api.model is a subset of api.multi

Reply:
Only inside your mind

Reply:
it isnt only inside your mind
```python
def model(method):
    """ Decorate a record-style method where ``self`` is a recordset, but its
        contents is not relevant, only the model is. Such a method::

            @api.model
            def method(self, args):
                ...

        may be called in both record and traditional styles, like::

            # recs = model.browse(cr, uid, ids, context)
            recs.method(args)

            model.method(cr, uid, args, context=context)

        Notice that no ``ids`` are passed to the method in the traditional style.
    """
```
the only difference is that no ids are passed, this is what is called a subset